### ISSUE TITLE
build, android: `make apk` fails if depends source cache is in a non-default location

### ISSUE THREAD

**Issue by achow101** [bitcoin/bitcoin#22522]
When building the Android dependencies and `SOURCES_PATH` is set to a non-default location (i.e. not in the depends directory), doing `make apk` results in the following error:

```
find: ‘../depends/sources/’: No such file or directory
find: ‘../depends/sources/’: No such file or directory
tar: --exclude=*/*: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
find: ‘../depends/sources/’: No such file or directory
find: ‘../depends/sources/’: No such file or directory
tar: --exclude=*/*: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
find: ‘../depends/sources/’: No such file or directory
find: ‘../depends/sources/’: No such file or directory
tar: --exclude=*/*: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

It seems like the culprit is https://github.com/bitcoin/bitcoin/blob/a3791da0e80ab35e862989373f033e5be4dff26b/src/Makefile.qt.include#L385-L387 which is looking for files in the default depends sources path.

**Comment by dongcarl** [comment]
Seems like `depends` should define `QT_BASE_PATH` in `config.site` instead?

**Comment by hebasto** [comment]
@achow101 @dongcarl 

Do you mind testing a fix in #22555?

**Comment by hebasto** [comment]
This can be closed as builds for Android are [not possible](https://github.com/bitcoin/bitcoin/issues/29360) in the master branch.
